docs/documentation/make-first-prototype/show-users-answers.md

```diff
@@ -1,55 +1,111 @@
-# Show the user's answers on your 'Check your answers' page
+# Show the user’s answers on the ‘Check your answers’ page
 
-If you do research with real user data, you must clear a user's data before each session by either:
+The Prototype Kit stores answers that users enter, and you can use them to make more realistic prototypes, for example by showing answers for users to check.
 
-- closing all browser windows and [opening a new incognito window](https://support.google.com/chrome/answer/95464)
-- selecting the **Clear data** link at the bottom of the prototype
+## Show the answer to question 1
 
-Check you’ve cleared the data by returning to a previously-loaded page and making sure the data is gone.
+1. Open `check-your-answers.html` in your `app/views` folder.
+2. Find the `<dt>` tag that contains the text 'Name'.
+3. Change 'Name' to 'Number of balls you can juggle'.
+4. In the `<dd>` tag on the next line, change 'Sarah Philips' to `{{ data['how-many-balls'] }}`.
```

Comment:
I know this line is probably just quoting what's in the UI, but isn't that surname usually spelled with two _l_s?